style.css

```diff
@@ -42,12 +43,14 @@ body {
   margin: 0;
   font-family: "Poppins", sans-serif;
   line-height: 1.5em;
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-template-columns: auto;
   color: #172b4d;
-  background-image: url("./header-shapes mobile.png");
+  background-image: url("./images/header-shapes mobile.svg");
```

Comment:
- Kindly make sure that you apply the background image on the home page and not in all containers , please fix that to make your project look like the figma design :+1:  as required in the project 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/227675317-b5b6596d-0f1a-4973-ba71-393573fed5dd.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/227675367-510d00ea-9c74-424b-b28d-202a065649aa.png)